### PR TITLE
Completed prep-javascript-conditional Exercise - Sanford Huynh

### PR DESCRIPTION
**Pull request by sanfordhuynh**
Conditionals

### CODE REVIEW

File: prep-javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    return 'A';
+  } else {
+    return 'A++';
+  }
+}
+console.log(gradeCalculator(student2Score));
+
+const season = 'winter';
+
+function seasonMessenger(seaosn) {
```
Comment (BrettAlbright) [inline prep-javascript-conditionals/main.js]: typo here, make sure the parameter is correctly spelled.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by sanfordhuynh** [comment]
revised

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**